ui/mirage/scenarios/default.js

```diff
@@ -11,6 +11,7 @@ const allScenarios = {
   massiveCluster,
   allJobTypes,
   allNodeTypes,
+  allTheThings,
```

Comment:
Although I love the name here, would it be better to call this something to do with screengrabs? Maybe like 'screengrabScenario'? I'm assuming this scenario is especially for taking the screengrabs? Maybe just a comment if you want to keep the name and potentially use it for other things also?

Reply:
I definitely made this one for the screengrabs, and the puppeteer script will break if this one isn't used, but I also think this scenario is valid beyond screengrabs.

I agree that the coupling needs to be communicated in some manner.